Expecto/Expecto.fs

```diff
@@ -574,19 +614,13 @@ module Impl =
     |> Seq.toList
 
   /// Evaluates tests sequentially
-  let evalSeq =
-    let printer =
-      TestPrinters.Default
-
+  let evalSeq printer =
```

Comment:
Without the change, custom printer is only used for `beforeRun` and `summary`, the rest of functions are from the `Default` printer